imports/plugins/included/payments-stripe/server/methods/stripe-connect.js

```diff
@@ -0,0 +1,73 @@
+import { Meteor } from "meteor/meteor";
+import { HTTP } from "meteor/http";
+import { check } from "meteor/check";
+import { Reaction, Logger } from "/server/api";
+import { Packages } from "/lib/collections";
+
+Meteor.methods({
```

Comment:
Good catch. We will definitely want to write tests and I had planned to do this.